contracts/interfaces/IContractIdentifier.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+// General interface for upgradable contracts
+interface IContractIdentifier {
+    /**
+     * @dev Returns the contract ID. It can be used as a check during upgrades.
+     * @notice Meant to be overridden in derived contracts.
```

Comment:
```suggestion
     * @notice Returns the contract ID. It can be used as a check during upgrades.
     * @dev Meant to be overridden in derived contracts.
```